module-2/lab-mongo-m2/lab-mongo-m2.ipynb

```diff
@@ -0,0 +1,1297 @@
+{
```

Comment:
- "Q11: All the companies founded on the second semester of the year. Limit your search to 1000 companies.". I think you are only including greater than 6, don't you? You should also include month 6. 
- "Q7: All the companies that have a null type of value on the category_code field". Null is not the same as None. I would query both separately. I encourage you to ask Marc the difference between Null and undefined in JavaScript.
-Remember to ask Marc to solve question number 12!!
- I invite you to explore the aggregate () function in mongo, similar to groupby. Play with this data set to get some aggregations from Mongodb.
- Try to complete all the queries of the exercise when you have some free time.